tests/conftest.py

```diff
@@ -71,12 +65,46 @@ def mock_product_test_spotlight(
     )
 
 
+@pytest.fixture
+def mock_product_test_wolf_cola(
+    product_backend: MockProductBackend, mock_provider: Provider
+) -> Product:
+    """Fixture for a mock Wolf Cola product."""
```

Comment:
I would prefer if we didn't put cheeky pop culture references in the tests. Just name it something generic to do with satellites.